scripts/build.sh

```diff
@@ -35,7 +35,7 @@ if [ -n "${BUILDTAGS}" ]; then
 fi
 
 export GOARCH="${ARCH}"
-export CGO_ENABLED=0
+export CGO_ENABLED=1
```

Comment:
This is not strictly necessary, however it's nice to be able to share the cgo consts in the loopback code, which is what this allows

Reply:
I did manage to implement it without cgo; I can switch back and just be a little unhappy about the consts.

Reply:
I removed the cgo dependency. I'm happy to squash idown those commits, but they're kept separate for now.